catalog/monitors/ec2-cpu-utilization.yaml

```diff
@@ -0,0 +1,36 @@
+# The official Datadog API documentation with available query parameters & alert types:
+# https://docs.datadoghq.com/api/v1/monitors/#create-a-monitor
+
+ec2-cpu-utilization:
+  name: (EC2) CPU Utilization on {{instance_id.name}}
```

Comment:
How `{{instance_id.name}}` will be interpolated in the `name`?
Will it be a separate monitor for each possible instance?

Maybe it should just be `name: (EC2) CPU Utilization`